Packs/SplunkPy/ReleaseNotes/3_2_4.md

```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### SplunkPy
+
+- Documentation and metadata improvements.
```

Comment:
```suggestion
Documentation and metadata improvements.
```

Reply:
It will not change because it is not related to the above contribution but to the commit from master.